V2/tests/integration/daos/time_entries_dao_test.py

```diff
@@ -69,3 +69,32 @@ def test_delete__returns_none__when_no_time_entry_matching_its_id_is_found(
     result = dao.delete(Faker().pyint())
 
     assert result is None
+
+
+def test_update__returns_an_time_entry_dto__when_found_one_time_entry_to_update(
+    test_db, create_fake_dao, time_entry_factory, insert_activity, activity_factory
+):
+    dao = create_fake_dao(test_db)
+    inserted_activity = insert_activity(activity_factory(), dao.db)
+    existent_time_entries = time_entry_factory(activity_id=inserted_activity.id, technologies="[jira,sql]")
```

Comment:
Perhaps is convenient not to send technologies param if we are not going to use it